### PR TITLE
BXMSDOC-5412 Append all employee-rostering artifacts with -optaweb

### PR DESCRIPTION
**Pull request by emmurphy1**
Details in jira
[BXMSDOC-5342](https://issues.redhat.com/browse/BXMSDOC-5342)

### CODE REVIEW

File: doc-content/enterprise-only/employee-rostering/optashift-ER-building-eclipse-proc.adoc [hunk 1]
```diff
@@ -40,32 +40,32 @@ You can change the name, location. and arguments to use another browser instead
...
-*** Working Directory: `${workspace_loc:/employee-rostering}`
+*** Working Directory: `${workspace_loc:/optaweb-employee-rostering-}`
 *** Arguments: `fuser -k 9876/tcp`
 +
 . From the main menu, select *Run* -> *Run Configurations...*.
 . Under *Maven Build*, create the following launch configurations:
 .. OptaWeb Employee Rostering Build:
 *** Name: `OptaWeb Employee Rostering Build`
-*** Base directory: `${workspace_loc:/employee-rostering}`
+*** Base directory: `${workspace_loc:/optaweb-employee-rostering-}`
```
Comment (Christopher-Chianelli) [inline doc-content/enterprise-only/employee-rostering/optashift-ER-building-eclipse-proc.adoc]:
```suggestion
*** Base directory: `${workspace_loc:/optaweb-employee-rostering}`
```

File: doc-content/enterprise-only/employee-rostering/optashift-ER-building-eclipse-proc.adoc [hunk 2]
```diff
@@ -40,32 +40,32 @@ You can change the name, location. and arguments to use another browser instead
...
 *** Goals: `clean install`
 *** Parameter: `gwt:skipCompilation` Value: `true`
 +
 .. OptaWeb Employee Rostering Start Code Server:
 *** Name: `OptaWeb Employee Rostering Start Code Server`
-*** Base directory: `${workspace_loc:/employee-rostering}`
+*** Base directory: `${workspace_loc:/optaweb-employee-rostering-}`
```
Comment (Christopher-Chianelli) [inline doc-content/enterprise-only/employee-rostering/optashift-ER-building-eclipse-proc.adoc]:
```suggestion
*** Base directory: `${workspace_loc:/optaweb-employee-rostering}`
```

File: doc-content/enterprise-only/employee-rostering/optashift-ER-building-eclipse-proc.adoc [hunk 3]
```diff
@@ -40,32 +40,32 @@ You can change the name, location. and arguments to use another browser instead
...
 *** Goals: `gwt:codeserver`
 *** Parameter: `gwt:skipCompilation` Value: `true`
 +
 .. OptaWeb Employee Rostering Start Webserver:
 *** Name: `OptaWeb Employee Rostering Start Webserver`
-*** Base directory: `${workspace_loc:/employee-rostering}`
+*** Base directory: `${workspace_loc:/optaweb-employee-rostering-}`
```
Comment (Christopher-Chianelli) [inline doc-content/enterprise-only/employee-rostering/optashift-ER-building-eclipse-proc.adoc]:
```suggestion
*** Base directory: `${workspace_loc:/optaweb-employee-rostering}`
```

File: doc-content/enterprise-only/employee-rostering/optashift-ER-building-eclipse-proc.adoc [hunk 4]
```diff
@@ -40,32 +40,32 @@ You can change the name, location. and arguments to use another browser instead
...
 *** Goals: `wildfly:start wildfly:deploy`
 *** Parameter: `gwt:skipCompilation` Value: `true`
 +
 .. OptaWeb Employee Rostering Stop Webserver:
 *** Name: `OptaWeb Employee Rostering Stop Webserver`
-*** Base directory: `${workspace_loc:/employee-rostering}`
+*** Base directory: `${workspace_loc:/optaweb-employee-rostering-}`
```
Comment (Christopher-Chianelli) [inline doc-content/enterprise-only/employee-rostering/optashift-ER-building-eclipse-proc.adoc]:
```suggestion
*** Base directory: `${workspace_loc:/optaweb-employee-rostering}`
```

File: doc-content/enterprise-only/employee-rostering/optashift-ER-modifying-proc.adoc [hunk 5]
```diff
@@ -21,7 +21,7 @@ The following procedure outlines the general approach to modifying the employee
 +
 . If any required data can be calculated from the current data or can be hardcoded, add the calculations or hardcoding to existing view or utility classes. If the data must be calculated on the server side, add REST API endpoints to read it.
 . If any required data must be entered by the user, add the data to the classes representing the data entities (for example, the `Employee` class), add REST API endpoints to read and write the data, and modify the user interface to enter the data.
-. When all the data is available, modify the rules. For most modifications, you must add a new rule. The rules are located in the `src/main/resources/org/optaweb/employeerostering/service/solver/employeeRosteringScoreRules.drl` file of the `employee-rostering-backend` module.
+. When all the data is available, modify the rules. For most modifications, you must add a new rule. The rules are located in the `src/main/resources/org/optaweb/employeerostering/service/solver/employeeRosteringScoreRules.drl` file of the `optaweb-employee-rostering--backend` module.
```
Comment (Christopher-Chianelli) [inline doc-content/enterprise-only/employee-rostering/optashift-ER-modifying-proc.adoc]:
```suggestion
. When all the data is available, modify the rules. For most modifications, you must add a new rule. The rules are located in the `src/main/resources/org/optaweb/employeerostering/service/solver/employeeRosteringScoreRules.drl` file of the `optaweb-employee-rostering-backend` module.
```

File: doc-content/enterprise-only/employee-rostering/optashift-ER-modifying-sample-weekend-proc.adoc [hunk 6]
```diff
@@ -1,14 +1,14 @@
...
+With this sample modification, the employee rostering starter application does not place people on a roster for two weekends in a row. If a person is rostered for a Saturday or Sunday shift, the next Saturday and Sunday are guaranteed to be free.
 
 The new rule relies on the information that is already available in the application. No changes to data structures or to the user interface are required. It is an example of a simple change in the starter application.
 
 You can review the full change and apply it by using the ??? patch file, available from ???
 
 .Procedure
 
-. Add the determination of whether the shift is a weekend shift to the `src/main/java/org/optaweb/employeerostering/shared/shift/Shift.java` class of the `employee-rostering-shared` package. Add the following method:
+. Add the determination of whether the shift is a weekend shift to the `src/main/java/org/optaweb/employeerostering/shared/shift/Shift.java` class of the `optaweb-employee-rostering--shared` package. Add the following method:
```
Comment (Christopher-Chianelli) [inline doc-content/enterprise-only/employee-rostering/optashift-ER-modifying-sample-weekend-proc.adoc]:
```suggestion
. Add the determination of whether the shift is a weekend shift to the `src/main/java/org/optaweb/employeerostering/shared/shift/Shift.java` class of the `optaweb-employee-rostering-shared` package. Add the following method:
```

File: doc-content/enterprise-only/employee-rostering/optashift-ER-runningwar-proc.adoc [hunk 7]
```diff
@@ -25,5 +25,5 @@ EAP_HOME/bin/jboss-cli.sh --connect
 ----
 +
 . Stop the application server.
-. Deploy the `binaries/employee-rostering-webapp-{MAVEN_ARTIFACT_VERSION}.war` directory into the application server and start the application server.
+. Deploy the `binaries/optaweb-employee-rostering-webapp-{MAVEN_ARTIFACT_VERSION}.war` directory into the application server and start the application server.
```
Comment (Christopher-Chianelli) [inline doc-content/enterprise-only/employee-rostering/optashift-ER-runningwar-proc.adoc:28]: I am pretty sure we don't supply a "WAR" file anymore (and the webapp module no longer exists).

**Review by Christopher-Chianelli — CHANGES_REQUESTED** [review]

File: doc-content/enterprise-only/employee-rostering/er-deploy-prepare-proc.adoc [hunk 8]
```diff
@@ -8,9 +8,9 @@ You must download and prepare the deployment files before building and deploying
 . Download the `{PRODUCT_FILE}-reference-implementation.zip` file from the {PRODUCT_DOWNLOAD_LINK}[Software Downloads] page for {PRODUCT} {PRODUCT_VERSION}.
 . Unzip the downloaded archive.
 . Copy the contents of the `jboss-rhba-{PRODUCT_VERSION_LONG}.GA-maven-repository/maven-repository` subdirectory into the `~/.m2/repository` directory.
-. Expand the `{PRODUCT_INIT}-{PRODUCT_VERSION}-employee-rostering.zip` file that is extracted from the reference implementation archive. 
+. Expand the `{PRODUCT_INIT}-{PRODUCT_VERSION}-optaweb-employee-rostering--.zip` file that is extracted from the reference implementation archive.
 +
-The `employee-rostering-distribution-{MAVEN_ARTIFACT_VERSION}` folder is created. This folder is the base folder in subsequent parts of this document.
+The `optaweb-employee-rostering--distribution-{MAVEN_ARTIFACT_VERSION}` folder is created. This folder is the base folder in subsequent parts of this document.
```
Comment (cuijulian) [inline doc-content/enterprise-only/employee-rostering/er-deploy-prepare-proc.adoc]: Should be `optaweb-employee-rostering-distribution-{MAVEN_ARTIFACT_VERSION}`

File: doc-content/enterprise-only/employee-rostering/optashift-ER-runningwar-proc.adoc (hunk 7 above)
Comment (emmurphy1) [reply doc-content/enterprise-only/employee-rostering/optashift-ER-runningwar-proc.adoc:28]: This file isn't used. It was updated as part of the global update.

**Review by Christopher-Chianelli — APPROVED** [review]
LGTM

**Review by cuijulian — APPROVED** [review]
LGTM.

File: doc-content/enterprise-only/employee-rostering/optashift-ER-building-IntelliJ-IDEA-proc.adoc [hunk 9]
```diff
@@ -13,10 +13,10 @@ You can use IntelliJ IDEA to build and run the employee rostering starter applic
 . Select the root directory of the application source and click *OK*.
 . From the main menu, select *Run* -> *Edit Configurations*.
 . In the window that appears, expand *Templates* and select *Maven*. The Maven sidebar appears.
-. In the Maven sidebar, select `employee-rostering-backend` from the  *Working Directory* menu.
+. In the Maven sidebar, select `optaweb-employee-rostering--backend` from the  *Working Directory* menu.
 . In *Command Line*, enter `spring-boot:run`.
 . To start the back end, click *OK* .
-. In a command terminal , navigate to the `employee-rostering-frontend` directory.
+. In a command terminal , navigate to the `optaweb-employee-rostering--frontend` directory.
```
[the remark this answers is not in the document]
Comment (michelehaglund) [reply doc-content/enterprise-only/employee-rostering/optashift-ER-building-IntelliJ-IDEA-proc.adoc]: @emmurphy1 - extra space after "terminal"

In a command terminal , navigate to the `optaweb-employee-rostering-frontend` directory.

**Review by michelehaglund — APPROVED** [review]
Looks good @emmurphy1 One minor comment.

**Pull request closed without merging**